kubetest/anywhere.go

```diff
@@ -199,3 +205,126 @@ func (k kubernetesAnywhere) Down() error {
 	}
 	return finishRunning(exec.Command("make", "-C", k.path, "FORCE_DESTROY=y", "destroy"))
 }
+
+const DefaultConfigFile = ".config"
+
+type kubernetesAnywhereMulti struct {
+	*kubernetesAnywhere
+	multiClusters multiClusterDeployment
+}
+
+var _ deployer = kubernetesAnywhereMulti{}
```

Comment:
Does this line do anything?

Reply:
This is to check the type implements the interface.

Reply:
Ah, makes sense (although it looks removed now).